.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "endOfLine": "auto"
```

Comment:
Il est en Lf par défaut depuis la version 2 du coup cela for la fin de fichier au format unix pour tous le monde pourquoi il faudrait le changer ?

Reply:
Perso je ne config plus git avec `core.autocrlf=true` je préfère mettre `core.autocrlf=input.`
L'avantage c'est que si je commit un fichier en fin de ligne Windows il est automatiquement convertie en fin de linux linux. Par contre au `git clone` ou `git chekout` il n'y a pas de transformation LF vers CRLF.
J'ai également configurer mon IDE pour que chaque nouveau fichier sois créer en fin de ligne LF.

D'ailleurs prettier recommande de mettre `* text=auto eol=lf` dans `.gitattributes` à la racine du dépôt histoire d'être sur.

https://prettier.io/docs/en/options#end-of-line

Reply:
la raison est simple, et tu l'as décrit dans ton message. Si tu veux avoir du lf partout, il faut faire plein de configs. Alors que dans le mode auto, un nouveau dev aura la même expérience sur windows linux mac ou autre.
Il faut baisser au maximum le ticket d'entrée pour contribuer. 
Le .gitattributes ça va forcer tlm à re-cloner le repo également. 
Tout ça  pour quoi exactement ? Ca n'apporte pas vraiment quelque chose :/

Reply:
Du coup peux tu fait comme j'ai fait pour es(style)lint
A savoir dans le dossier config un fichier prettier.config.js dans dans les packages un .prettierrc et require la config
histoire de rester cohérent

Reply:
c'est fait